docker-compose.yml

```diff
@@ -81,18 +82,20 @@ services:
     - -c 
     - mkdir -p /data/scholarsphere; minio --compat server --address ':9000' /data
   solr:
-    image: psul/solr:8.5.2-slim
+    image: solr:8.11.1-slim
     restart: always
     environment:
-      SOLR_STOP_WAIT: 5
+      SOLR_STOP_WAIT: 10
```

Comment:
Just for my own info, what is the significance of this environment variable and why did it change?

Reply:
SOLR_STOP_WAIT is the time solr will wait before forcefully killing it. it defaults to something crazy like 180. as it turns out that variable is also used for the start_wait time, and i had to increase it.